diff -uNr linux-source-5.4-orig/drivers/i2c/busses/i2c-isch.c linux-source-5.4/drivers/i2c/busses/i2c-isch.c
--- linux-source-5.4-orig/drivers/i2c/busses/i2c-isch.c	2020-02-11 13:35:55.000000000 +0100
+++ linux-source-5.4/drivers/i2c/busses/i2c-isch.c	2020-04-09 22:50:13.934371452 +0200
@@ -44,6 +44,8 @@
 #define SCH_WORD_DATA		0x03
 #define SCH_BLOCK_DATA		0x05
 
+#define SMBIOSIZE	8
+
 static unsigned short sch_smba;
 static struct i2c_adapter sch_adapter;
 static int backbone_speed = 33000; /* backbone speed in kHz */
@@ -262,7 +264,7 @@
 	if (!res)
 		return -EBUSY;
 
-	if (!devm_request_region(&dev->dev, res->start, resource_size(res),
+	if (!devm_request_region(&dev->dev, res->start, SMBIOSIZE,
 				 dev->name)) {
 		dev_err(&dev->dev, "SMBus region 0x%x already in use!\n",
 			sch_smba);
