diff -uNr linux-source-5.4-orig/drivers/gpu/drm/gma500/backlight.c linux-source-5.4/drivers/gpu/drm/gma500/backlight.c
--- linux-source-5.4-orig/drivers/gpu/drm/gma500/backlight.c	2020-02-11 13:35:55.000000000 +0100
+++ linux-source-5.4/drivers/gpu/drm/gma500/backlight.c	2020-04-09 22:54:06.125076646 +0200
@@ -13,6 +13,8 @@
 #include "intel_bios.h"
 #include "power.h"
 
+#undef CONFIG_BACKLIGHT_CLASS_DEVICE  // OpenFrame Workaround
+
 #ifdef CONFIG_BACKLIGHT_CLASS_DEVICE
 static void do_gma_backlight_set(struct drm_device *dev)
 {
diff -uNr linux-source-5.4-orig/drivers/gpu/drm/gma500/psb_device.c linux-source-5.4/drivers/gpu/drm/gma500/psb_device.c
--- linux-source-5.4-orig/drivers/gpu/drm/gma500/psb_device.c	2020-02-11 13:35:55.000000000 +0100
+++ linux-source-5.4/drivers/gpu/drm/gma500/psb_device.c	2020-04-09 22:54:06.125076646 +0200
@@ -63,6 +63,8 @@
 	uint32_t value;
 	uint32_t blc_pwm_precision_factor;
 
+	return 0;  // OpenFrame Workaround
+
 	/* get bl_max_freq and pol from dev_priv*/
 	if (!dev_priv->lvds_bl) {
 		dev_err(dev->dev, "Has no valid LVDS backlight info\n");
